dev-games/flatzebra/flatzebra-0.1.6.recipe

```diff
@@ -0,0 +1,62 @@
+DESCRIPTION="A generic game engine for 2D double-buffering animation"
+SUMMARY="Generic game engine"
+HOMEPAGE="http://sarrazip.com/dev/burgerspace.html"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2009 Pierre Sarrazin."
+SOURCE_URI="http://perso.b2b2c.ca/sarrazip/dev/flatzebra-$portVersion.tar.gz"
+CHECKSUM_SHA256="4e7bb0a77136ec3b81e0f73c1d08e828d38ef011095d5ce7068a94f3bb21d67a"
+REVISION="2"
+SOURCE_DIR="flatzebra-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
```

Comment:
Sorry, but according to the wiki guidelines there should be blank line after ARCHITCTURES segment.
https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines